# Marker coordinates (from the marker "tibia_body")
femur_body:
- x: 0.003953
  y: 0.013058
  z: 0.000468
- x: -0.014963
  y: 0.010491
  z: -0.002972
- x: 0.019219
  y: -0.002378
  z: 0.002969
- x: -0.000544
  y: -0.009617
  z: -0.018597
- x: -0.007665
  y: -0.011554
  z: 0.018132
femur_ref:
- x: 0.0
  y: 0.0
  z: -0.0254
- x: 0.0
  y: -0.0176
  z: -0.018
- x: 0.0
  y: 0.0194
  z: -0.018
- x: 0.0185
  y: 0.0
  z: 0.0
- x: -0.02
  y: 0.0
  z: 0.0
tibia_body:
- x: -0.010117
  y: 0.012947
  z: -0.010044
- x: 0.012788
  y: 0.01289
  z: -0.009797
- x: 0.020272
  y: -0.011399
  z: 0.004265
- x: -0.01179
  y: -0.019203
  z: -0.012112
- x: -0.011153
  y: 0.004765
  z: 0.027688
tibia_ref:
- x: 0.0
  y: -0.0176
  z: -0.02
- x: 0.0
  y: 0.0
  z: -0.0354
- x: 0.0
  y: 0.0234
  z: -0.02
- x: 0.018
  y: 0.0
  z: 0.0
- x: -0.028
  y: 0.0
  z: 0.0
